### ISSUE TITLE
training without ImageNet datasets hang for 10min in the beginning

### ISSUE THREAD

**Issue by simonschuang** [myelintek/web#14]

**Comment by anastasiia-annata** [comment]
I train GoogleNet on ImageNet on workstation 140.96.29.110. I don't see this problem. I train inside myelintek/web docker container.

2018-03-20 **05:55:24** [20180320-055522-055c] [INFO ] Train Tensorflow Model task started.
2018-03-20 **05:56:53** [20180320-055522-055c] [DEBUG] Validation (epoch 0): loss = 7.186995, accuracy = 0.000996
2018-03-20 **05:57:04** [20180320-055522-055c] [DEBUG] Training (epoch 0.004): loss = 6.962795, lr = 0.100000, accuracy = 0.000781

**Comment by anastasiia-annata** [comment]
For comparing. Train GoogleNet on ImageNet on DGX-1 with DIGITS, inside NV-cloud container. Train on 8*V100 GPU, batch size is 4096. Disk looks like hdd.

2018-03-19 **16:42:47** [INFO] Train batch size is 4096 and validation batch size is 4096
2018-03-19 **16:44:33** [INFO] Validation (epoch 0): loss = 7.199092, accuracy = 0.000939
2018-03-19 **16:44:49** [INFO] Training (epoch 0.006): loss = 7.130652, lr = 0.400000, accuracy = 0.001221

Full log attached.
[tensorflow_output.log](https://github.com/myelintek/web/files/1828152/tensorflow_output.log)

**Comment by simonschuang** [comment]
It seems the problem happens on VGG network. This is the log on our server with 1080Ti * 8

2018-03-20 07:45:24 [INFO] Train batch size is 32 and validation batch size is 32
2018-03-20 07:46:12 [INFO] Starting queue runners (val)
2018-03-20 **07:46:12** [INFO] Queue runners started (val)
2018-03-20 **07:56:43** [INFO] Validation (epoch 0): loss = 11.080255, accuracy = 0.000920
2018-03-20 07:56:43 [INFO] During training. details will be logged after every 157 steps (batches)
2018-03-20 07:56:43 [INFO] While logging, epoch value will be rounded to 3 significant digits
2018-03-20 07:56:43 [INFO] Starting queue runners (train)
2018-03-20 07:56:43 [INFO] Queue runners started (train)
2018-03-20 07:56:43 [INFO] Started training the model

**Comment by anastasiia-annata** [comment]
I tried to run VGG on 4*1080 server (workstation 140.96.29.110, same as first comment). Butch size is 32.
2018-03-20 **07:18:49** [20180320-071848-b33d] [INFO ] Train Tensorflow Model task started.
2018-03-20 **07:23:42** [20180320-071848-b33d] [DEBUG] Validation (epoch 0): loss = 11.070996, accuracy = 0.001400
2018-03-20 **07:24:41** [20180320-071848-b33d] [DEBUG] Training 0.01% complete.
2018-03-20 07:24:41 [20180320-071848-b33d] [DEBUG] Training (epoch 0.004): loss = 11.067138, lr = 0.010000, accuracy = 0.001393

**Comment by anastasiia-annata** [comment]
Same, but using one 1080 GPU.
2018-03-20 **08:29:40** [20180320-082939-e97d] [INFO ] Train Tensorflow Model task started.
2018-03-20 **08:34:11** [20180320-082939-e97d] [DEBUG] Network validation loss #0.0: 11.085127

**Comment by simonschuang** [comment]
Okay, I'm not sure why it depends on # of GPU, here is the VGG network with 1080Ti * 4

2018-03-20 08:34:36 [INFO] Starting queue runners (val)
2018-03-20 08:34:36 [INFO] Queue runners started (val)
2018-03-20 08:36:36 [INFO] Validation (epoch 0): loss = 11.082155, accuracy = 0.000999
2018-03-20 08:36:36 [INFO] During training. details will be logged after every 40 steps (batches)
2018-03-20 08:36:36 [INFO] While logging, epoch value will be rounded to 3 significant digits
2018-03-20 08:36:36 [INFO] Starting queue runners (train)
2018-03-20 08:36:37 [INFO] Queue runners started (train)
2018-03-20 08:36:37 [INFO] Started training the model
